Sources/Commands/SwiftTestTool.swift

```diff
@@ -217,7 +299,9 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                 processSet: processSet,
                 sanitizers: options.sanitizers,
                 toolchain: toolchain,
-                xUnitOutput: options.xUnitOutput
+                xUnitOutput: options.xUnitOutput,
+                numJobs: options.numberOfWorkers != nil ? options.numberOfWorkers! : ProcessInfo.processInfo.activeProcessorCount,
```

Comment:
You can use nil-coalescing operator: `options.numberOfWorkers ?? ProcessInfo.processInfo.activeProcessorCount`